pages/docs/concepts/asyncapi-document/dynamic_channel_names.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Dynamic Channel Names
+weight: 220
+---
+
```

Comment:
Please fix your file name to replace underscores with dashes: `dynamic-channel-names.md`.

Reply:
Pretty well-written and informative! ✨✨ but could benefit from additional context and organization. ✌🏽

---

1. **Introduction**: The introduction could be expanded to give a bit more background on the importance and advantages of using dynamic channel names in AsyncAPI. Could you mention **why the feature is used** and **how it can benefit folks or systems**?

2. **Explanation**: Some parts of the document could use a more detailed explanation. For example, when introducing the concept of _Parameter Context_, you could explain why it is important and how it aids in the developer experience.

3. **Organization**: You might want to reorganize the document a bit to improve the flow. For instance, before diving into 'Parameter Context', it might be better to introduce 'Reusing Parameters' since it directly follows the initial introduction of parameters in channels.

4. **Graphics**: Ensure your diagrams accurately represent the text as _clearly as possible_. The second diagram starting on line 57, for example, might be clearer if it included the word `payload`, to match what you wrote it in the text description introducing the diagram (i.e., "runtime event payload").

5. **Proofreading**: There are grammatical/spelling errors (like 'vesion' instead of 'version' in one of the YAML examples on line 81) that should be corrected. Please review this doc and fix those mistakes. 😄 

6. **Do not use dashes `-` instead of colons `:` at the end of a sentence.**